php-checks/src/main/java/org/sonar/php/checks/utils/CheckUtils.java

```diff
@@ -175,5 +190,72 @@ private static void appendChild(StringBuilder sb, @Nullable SyntaxToken prevToke
   private static boolean isSpaceRequired(SyntaxToken prevToken, SyntaxToken token) {
     return (token.line() > prevToken.line()) || (prevToken.column() + prevToken.text().length() < token.column());
   }
+  
+  public static List<Kind> CONTAIN_STATEMENTS = ImmutableList.of(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make CONTAIN_STATEMENTS a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this "public static CONTAIN_STATEMENTS" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this field "CONTAIN_STATEMENTS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS3008)